attribute/genericattribute.go

```diff
@@ -49,12 +51,33 @@ func NewGeneric(a *yotiprotoattr_v3.Attribute) *GenericAttribute {
 		value = a.Value
 	}
 
+	parsedAnchors := anchor.ParseAnchors(a.Anchors)
+
 	return &GenericAttribute{
 		Attribute: &yotiprotoattr_v3.Attribute{
 			Name:        a.Name,
 			ContentType: a.ContentType,
 		},
-		Value:   value,
-		Anchors: anchor.ParseAnchors(a.Anchors),
+		Value:     value,
```

Comment:
I'm not clear how this works in Go, but we need to sure that _Value_ is immutable too.  We don't really want people to have any ability to overwrite the values on the attributes we return.